bittensor/core/extrinsics/async_registration.py

```diff
@@ -212,154 +254,17 @@ async def register_extrinsic(
                 # Exited loop because pow is no longer valid.
                 logging.error("<red>POW is stale.</red>")
                 # Try again.
-                continue
+                # continue
 
+        # TODO(roman): discuss with team (looks like this section is not reachable)
```

Comment:
This can definitely be reached if the POW result is valid, but registration fails (and not because it's already registered) until the result goes stale.